tests/test_vcs_axisconvert_2d.py

```diff
@@ -55,9 +70,12 @@ def test_areawt(self):
                 vcs.createisofill,
                 vcs.createisoline,
                 vcs.createmeshfill,
-        #        vcs.createvector,
+                vcs.createvector,
+                vcs.createstreamline,
                        ]:
             self.axisConvertGmAreaWt(method)
             if not method in [vcs.createmeshfill,]:
-                self.axisConvertGmLog10(method)
-                self.axisConvertGmLog10Areawt(method)
+                self.axisConvertGmLinear(method)
+                if not method is vcs.createstreamline:
+                    self.axisConvertGmLog10(method)
```

Comment:
@danlipsa @scottwittenburg not log10 for streamlines? I guess it doesn't make much sense anyway. But just wondering.

Reply:
In general it should be fine whenever log10 scaling the axis makes sense, it just didn't seem to make sense with the test data sets (we can't do log10 on an axis when there are negative values there, and it doesn't make sense when the two variables are the same).  Is there a better pair of variables for this particular test case?  Same situation for vectors really I guess.